components/count-badge/count-badge-mixin.js

```diff
@@ -156,25 +157,29 @@ export const CountBadgeMixin = superclass => class extends LocalizeCoreElement(S
 
 	connectedCallback() {
 		super.connectedCallback();
-		if (!this.maxDigits && this.type === 'notification') {
-			// default to two digits for notification type
-			this.maxDigits = 2;
+		if (!this.maxDigits) {
+			// default to two digits for notification type, 5 for count
+			this.maxDigits = this.type === 'notification' ? 2 : 5;
+		} else if (this.maxDigits > 5) {
+			// limit all badges to 5 digits
+			this.maxDigits = 5;
```

Comment:
I added the limit in both connectedCallback and updated so that the limit is enforced from the start for elements that never update (like content tool), and for demo pages where you can update the maxDigits manually.